README.md

```diff
@@ -47,69 +61,99 @@ About 90+% winrate against [Ai Ai](http://mrraow.com/index.php/aiai-home/aiai/)
 
 ![Sample game](botanik/sample_game.gif)
 
-
 ### Small World
 
 * [x] Support of [Small World](https://boardgamegeek.com/boardgame/40692/small-world) with 2 to 5 players
-* [x] All types of people and power from the original game are supported. Each turn of a player may need several actions (see code for more details).
+* [x] All types of people and power from the original game are supported. Each turn of a player may need several
+  actions (see code for more details).
 * [x] Extra effort to improve pretrained model for 2 players
 
-![Sample game with 2 players](smallworld/sample_game.gif) (Scores are on the 2 last rows after _sc=_, and number after _#_ is number of next round).
+![Sample game with 2 players](smallworld/sample_game.gif) (Scores are on the 2 last rows after _sc=_, and number after
+_#_ is number of next round).
 
 
 ---
 
-## Technical details
+## Installation
 
-To change game, change the import in `main.py` and in `pit.py` (also in `GenericNNetWrapper.py` for debug)
+Contrary to the repo this repo is based on, this one doesn't set up a Docker. Just launch the Python scripts from your
+PC, no special GPU support required.
 
-<details>
-  <summary>Click here for details about training, running or playing</summary>
+### Dependencies
+
+```
+pip3 install onnxruntime numba tqdm colorama coloredlogs
+pip3 install torch --extra-index-url https://download.pytorch.org/whl/cpu
+pip3 install torchvision onnx
```

Comment:
torchvision should be added on previous line instead.
And `onnx` isn't needed, just `onnxruntime`